notebooks/official/model_evaluation/custom_tabular_regression_model_evaluation.ipynb

```diff
@@ -0,0 +1,2261 @@
+{
```

Comment:
<p>Prediction, you get is <strong>"prediction": [0.522156954]</strong>, which is an array.</p><p> </p><p>Can you clarify what this line means?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/GoogleCloudPlatform/vertex-ai-samples/pull/942/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='71'/>

Reply:
<p>What I meant was "prediction" type is an array</p>

Reply:
<p>the output which we get is of type "list"</p>